web/src/favicon.ts

```diff
@@ -62,6 +64,9 @@ export function update_favicon(new_message_count, pm_count) {
 
         load_and_set_favicon(rendered_favicon);
     } catch (error) {
+        if (!(error instanceof Error)) {
+            throw error;
+        }
```

Comment:
An unrelated note here. The try-catch block was added in #18466 for fixing #18374.

It would be great to add a comment here referring to #18374 to explain that we need this check because we know very little about what this error *would* be.